docs/README.md

```diff
@@ -8,6 +8,12 @@ Jsonnet library for generating Drone CI configuration file.
 jb install github.com/crdsonnet/drone-libsonnet@master
 ```
 
+in the same directory, make sure you install all the necessary dependencies by running:
```

Comment:
Don't think this is necessary for usage, only for generating the docs. Perhaps we can just include that in the Makefile target?

Reply:
good point